components/Places/PlaceForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback, useState } from "react";
+import { ScrollView, Text, TextInput, View, StyleSheet } from "react-native";
+import { Colors } from "../../constants/Colors";
+import Button from "../UI/Button";
+import ImagePicker from "./ImagePicker";
+import LocationPicker from "./LocationPicker";
+import { Place } from "../../models/place";
+
+function PlaceForm({ onSavePlace }) {
+  const [enteredTitle, setEnteredTitle] = useState();
+  const [selectedImage, setSelectedImage] = useState();
+  const [pickedLocation, setPickedLocation] = useState();
+
+  function changeTitleHandler(text) {
+    setEnteredTitle(text);
+  }
+  function takeImageHAndler(imageUri) {
+    setSelectedImage(imageUri);
+  }
+  const pickLocationHandler = useCallback((location) => {
+    setPickedLocation(location);
+  }, []);
+
+  function savePlaceHandler() {
+    const placeData = new Place(enteredTitle, selectedImage, pickedLocation);
```

Comment:
check the values first to make sure nothing is null or undefined, if so to alert the user